src/lipalightninglib.udl

```diff
@@ -437,7 +437,8 @@ interface OfferKind {
 
 enum OfferStatus {
     "READY",
-    "FAILED",
+    "FAILED", // Claiming the offer failed, but it can be retried.
+    "REFUNDED", // The offer was not claimed in time, so the user got refunded (Pocket: Refund happened over the Fiat rails).
```

Comment:
That the offer was not claimed in time is only one of a few reasons why an offer got refunded. A full list of reasons for Pocket can be found [here](https://pocketbitcoin.com/developers/docs/rest/v1/webhooks#refund-reasons).

Reply:
Awesome, thanks!